aurvote

```diff
@@ -219,7 +219,14 @@ fi
 
 COOKIE_FILE=${COOKIE_FILE:-"$AV_TMP/cookies"}
 pkgnames=("${pkgnames[@]%/*}") # compatibility with yaourt <= 1.2.1
-[[ ! $pkgnames ]] && usage && exit 1
+
+if [[ ! $pkgnames ]]; then
+    if (( ACTION=="check" )); then
```

Comment:
`(( ... ))` is for arithmetic expressions. ``[[ $ACTION = "check" ]]`` must be used here.